### ISSUE TITLE
Loinc 2342-4:  "Glucose [Mass/volume] in Cerebral spinal fluid"

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#3615]
Suggest creating a new term for Loinc: 2342-4 [Glucose [Mass/volume] in Cerebral spinal fluid] 
New term label: Abnormal CSF glucose level
Child of 'Abnormality of the cerebrospinal fluid'

New term comment (if any):
Need increased and decreased terms as well

Your biocurator ID for loinc2hpo (if desired): HPO:nvasilevsky

**Comment by pnrobinson** [comment]
Note that reduced glucose in CSF is called Hypoglycorrhachia
https://hpo.jax.org/app/browse/term/HP:0011972

**Comment by pnrobinson** [comment]
I will add Hyperglycorrhachia and the parent class

**Comment by pnrobinson** [comment]
Hypoglycorrhachia HP:0011972
Hyperglycorrhachia HP_0031885
Abnormal CSF glucose level HP_0031884

**Comment by pnrobinson** [comment]
see https://github.com/TheJacksonLaboratory/loinc2hpoAnnotation/issues/12
